ee/clickhouse/models/test/test_cohort.py

```diff
@@ -339,6 +340,28 @@ def test_cohortpeople_basic(self):
         )
         self.assertEqual(len(results), 2)
 
+    @patch("posthog.tasks.calculate_cohort.calculate_cohort")
+    def test_cohortpeople_basic_paginating(self, mock_calculate_people):
+        for i in range(50):
+            Person.objects.create(
```

Comment:
Non-blocking and out of scope as I guess we're just maintaining consistency with other tests around this.

Is it possible to use `update_or_create_person` here from the test utils, thereby not directly depending on the django model interface? Or perhaps a relevant API call that will be more stable than this interface. That would free us up to, say, completely switch how we store people and just focus on the interface.

I would love to just have a test, for instance, that:

```
create_people_api_thats_abstract_from_storage_details
create_cohort_api_thats_abstract_from_storage_details()
poll_get_cohort_api_to_check_when_people_have_been_determined()
people = get_cohort_people()
assert ...
```

Anyway, not important but just want to make sure our tests are written against stable interfaces so we can more easily change underlying implementations.

Reply:
(although maybe that doesn't work with the pagination details, not sure 🤔 )